### ISSUE TITLE
Password protected swagger ui

### ISSUE THREAD

**Issue by notflorian** [KelkooGroup/theGardener#666]
The swagger ui should not be accessible, exept for the theGardner administrators.

Can you please add a Basic authentication, with a single login / password as a first step to make it slightly more complicated to hack?

**Comment by GeReinhart** [comment]
What about having 
- a passphrase stored in the application.conf
- request this passphrase on every endpoints that change the database
@notflorian  ?

**Comment by notflorian** [comment]
yes perfect, and also protect the swagger-ui with the same user/password
